config.yml

```diff
@@ -48,6 +43,9 @@ defaults:
       secretAccessKey:  !env AWS_SECRET_ACCESS_KEY
       region:           us-west-2
       apiVersion:       2014-01-01
+  monitor:
+    mock:             true
```

Comment:
It seems to go both ways (I copied this from another service).  I'll add this to my list for best practices.